libsource/src/main/java/com/virtusize/libsource/Virtusize.kt

```diff
@@ -77,6 +78,15 @@ class Virtusize(
                         virtusizeRepository.switchInPageRecommendation(SizeRecommendationType.compareProduct)
                     }
                 }
+                VirtusizeEvents.UserDeletedProduct.getEventName() -> {
```

Comment:
@akueisara if Aoyama passes the `id` of the deleted user product, would that be easier? That way, you would only need to run the fetch again if the deleted user product was the `selectedUserProduct`. WDYT?

Reply:
@akueisara it's not too difficult for frontend to make this change so if that would be easier I'm happy to do it

Reply:
@wwaterman12 I think it's a good idea. Let's go for it. 👍

Reply:
I think I declared a variable to store the user product list globally in the VirtusizeRepository class. I might be able to just delete it by the id from the list instead of fetching it from the server to save service costs. There are some duplicate API calls between SDK and the web view.